tests/testthat/test-convGDX2mif.R

```diff
@@ -16,48 +16,18 @@ test_that("Test if REMIND reporting is produced as it should and check data inte
   # add GDXs for comparison here:
   gdxPaths <- NULL
 
-  if (length(gdxPaths) == 0) {
-    defaultGdxPath <- file.path(tempdir(), "fulldata.gdx")
-    if (!file.exists(defaultGdxPath)) {
-      utils::download.file("https://rse.pik-potsdam.de/data/example/remind2_test-convGDX2MIF_fulldata.gdx",
-        defaultGdxPath,
-        mode = "wb", quiet = TRUE
-      )
-    }
-    gdxPaths <- defaultGdxPath
-  }
-
-  # finds for each AMT scenario the most recent, successfully converged GDX,
-  # that is no older than 30 days
-  .findAMTgdx <- function(gdxPaths = NULL, scenario = NULL) {
```

Comment:
Do you agree that we dont need this functio anmore, because the AMT gdxes will be uploaded to the RSE server by the AMT script?

Reply:
Fine for me. I used it sometimes, but I guess nobody else did. And as running `make build` on the cluster does not really work anyway at the moment, you can delete it.